src/mongo_logger.ts

```diff
@@ -217,10 +217,10 @@ export function createStdioLogger(stream: {
   write: NodeJS.WriteStream['write'];
 }): MongoDBLogWritable {
   return {
-    write: (log: Log): unknown => {
-      stream.write(inspect(log, { compact: true, breakLength: Infinity }), 'utf-8');
+    write: promisify((log: Log, cb: () => void): unknown => {
```

Comment:
Is there a reason for using `promisify` here and not just making this an `async` function?

Reply:
The Node.js streams interface expects a callback, right?